index.html

```diff
@@ -6,6 +6,37 @@
     <title>Les 3 - Opdracht CSS Grid</title>
 </head>
 <body>
+<div class="grid-container">
+    <header class="grid-item grid-header">
+        Grid block A
+    </header>
+    <content class="grid-item grid-content-b">
+        Lorem ipsum dolor sit amet, consectetur adipisicing elit. Aut eligendi eos esse est exercitationem inventore,
+        itaque libero modi necessitatibus neque nisi perferendis praesentium qui quia quisquam ut, vel. Culpa deserunt
+        dicta dolorem, dolores ducimus et explicabo illum in incidunt iste, labore magnam mollitia natus nesciunt
+        officiis possimus sed sunt totam ut voluptas. Cupiditate dolorum esse est exercitationem impedit natus nesciunt
+        nobis obcaecati, pariatur perspiciatis quia quibusdam quidem recusandae sit ut? Accusamus, asperiores
+        dignissimos error, eum fuga fugit laboriosam neque nostrum porro rem tempora vel veritatis. Consequatur
+        cupiditate ea ipsa iste molestias quibusdam quos rerum, sed! Assumenda corporis, dolore eaque fugit illo iusto
+        labore laudantium maxime necessitatibus obcaecati quibusdam quos rerum sapiente similique, sunt vero voluptate.
+        Ad adipisci alias aut commodi dicta dolore dolores error expedita, facilis in laboriosam mollitia nemo non
+        numquam odio officia provident quas quasi, quo, quod rem repudiandae sapiente similique tempora tempore tenetur
+        vel voluptatem. Beatae, voluptas!
+    </content>
+    <content class="grid-item grid-content-c">
+        Grid block C
+    </content>
+    <aside class="grid-item grid-aside-d">
+        Grid block D
+    </aside>
+    <aside class="grid-item grid-aside-e">
+        Grid block E
+    </aside>
+    <footer class="grid-item grid-footer">
+        Grid block F
+    </footer>
+
```

Comment:
Ik zou de regelcode op 1 regel zetten, dit is overzichtelijker.
Zelf heb ik ook gekozen voor a-side terwijl in de uitwerkingen de regelcode in <div> staat, uiteindelijk is dit wel overzichtelijker en makkelijker tenzij je andere waardes aan je items wilt geven.